tools/make_patch.c

```diff
@@ -394,28 +419,22 @@ void interpret_command(
 	}
 }
 
-struct Patch *process_template(
+struct Patches *process_template(
 	const char *template_filename, FILE *new_rom, FILE *orig_rom,
 	const char *output_filename, const struct Symbol *symbols
 ) {
 	FILE *file = xfopen(template_filename, 'r');
 	FILE *output = xfopen(output_filename, 'w');
 
-	struct Patch *patches = create_buffer();
-	struct Patch *patch = patches;
+	struct Patches *patches = create_patches();
 
-	// The ROM checksum will always differ
-	patch->offset = 0x14e;
-	patch->size = 2;
-	patch++;
-	// The Stadium checksum will always differ
-	patch->offset = 0x1FFDE0;
-	patch->size = 0xFFFFFF - 0x1FFDE0;
-	patch++;
+	// The ROM checksum and Stadium data will always differ
+	append_patch(patches, 0x14e, 2);
+	append_patch(patches, 0x1ffde0, 0xffffff - 0x1ffde0);
```

Comment:
To match stadium.c:
```c
unsigned int rom_size = (unsigned int)xfsize("", orig_rom);
unsigned int patch_size = 24 + 6 + 2 + (rom_size / 0x2000) * 2;
append_patch(patches, rom_size - patch_size, patch_size);
```

Reply:
Resolved in https://github.com/pret/pokecrystal/pull/882/commits/2b20d3cd8c9b8cd45be67254dbd846e53c897946